recipe/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "torcheval" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: a498dec34137bc66c9cf1adc7353a46c604dd62255884c72dcb4e2e4fc2cd7e9
+    folder: pypi
+  - git_url: git@github.com:pytorch/torcheval.git
+    git_rev: f1bc22fc67ec2c77ee519aa4af8079f4fdaa41bb
+    git_depth: 1
+    folder: gitproj
+
+build:
+  skip: True  # [py<38]
+  number: 0
+  script:
+    - copy gitproj\*requirements.txt pypi  # [win]
+    - cp gitproj/*requirements.txt pypi    # [not win]
+    - cd pypi
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Reply:
I decided to keep the directory so the two sources are guaranteed to not overwrite each other.